<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>620</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <property name="elideMode">
   <enum>Qt::ElideMiddle</enum>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>顯示資料</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>資料庫資料查詢</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>541</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="P1_check">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>6</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>查詢</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="P1_combobox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="maxVisibleItems">
         <number>21</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>LAB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Teacher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Student</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Equipment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MOST_Project</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Execute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Accounting_of</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline;&quot;&gt;成大工科所員工及財產系統&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>查詢資料</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>實驗室人數&amp;應付薪水</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QPushButton" name="P2_check">
        <property name="text">
         <string>查詢</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="P2_tableWidget"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>實驗室</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0">
       <widget class="QPushButton" name="P2_project">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>有負責計畫的LAB</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="P2_student">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>已完成計畫的學生</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="P2_PhD">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>沒有碩士生的LAB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTableWidget" name="P2_tableWidget_4"/>
      </item>
      <item row="0" column="2">
       <widget class="QTableWidget" name="P2_tableWidget_3"/>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="P2_tableWidget_2"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>實驗室及老師擁有的經費(科技部計畫)</string>
    </property>
    <widget class="QTextEdit" name="textEdit_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;LAB_No.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="P2_tableWidget_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLineEdit" name="P2_No"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="P2_check_2">
        <property name="text">
         <string>查詢</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>300</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>實驗室設備的花費</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>29</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="1">
       <widget class="QPushButton" name="P2_avg">
        <property name="text">
         <string>平均花費</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="P2_maxmin">
        <property name="text">
         <string>最多與最少花費的設備</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="P2_tableWidget_6"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>更新資料</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>841</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>設備購入</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>361</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="4">
       <widget class="QTextEdit" name="textEdit_4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;LAB&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="P3_lab"/>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QTextEdit" name="textEdit_5">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;設備No.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QTextEdit" name="textEdit_6">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="P3_No">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="P3_date"/>
      </item>
      <item row="1" column="2">
       <widget class="QTextEdit" name="textEdit_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Cost&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTextEdit" name="textEdit_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Date&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="QLineEdit" name="P3_No_2"/>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="textEdit">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;設備No.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="P3_cost"/>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QPushButton" name="P3_delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QPushButton" name="P3_insert">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>841</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>加薪</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>361</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLineEdit" name="P3_salaryadd"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="P3_checkBox">
        <property name="text">
         <string>會計</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTextEdit" name="textEdit_7">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;增加多少薪水&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="P3_checkBox_2">
        <property name="text">
         <string>科技部計畫負責人</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="P3_salary">
        <property name="text">
         <string>確定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>查詢資料(Query)</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>731</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Query</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>481</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QLineEdit" name="P4_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="P4_check">
        <property name="text">
         <string>查詢</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="P4_tableWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>481</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
